tsconfig.json

```diff
@@ -17,7 +17,7 @@
       "chain-testing/*": ["./packages/chain-events/chain-testing/*"],
       "token-balance-cache": ["./packages/token-balance-cache"],
       "token-balance-cache/*": ["./packages/token-balance-cache/*"],
-      "react": ["node_modules/@types/react"]
+      "react": ["./node_modules/@types/react"]
```

Comment:
This change is a little scary -- any reason why it's needed? Otherwise, this can go in whenever.

Reply:
Yes so basically if a child `tsconfig.json` (like `packages/chain-events/tsconfig.json`) does not define a `baseUrl` then `yarn check-types` will fail with this error:
```
error TS5090: Non-relative paths are not allowed when 'baseUrl' is not set. Did you forget a leading './'?
```

`packages/commonwealth/tsconfig.json` defines `baseUrl: '.'` which allows `yarn --cwd packages/commonwealth check-types` to pass but `packages/chain-events/tsconfig.json` does not define `baseUrl` thus `yarn --cwd packages/chain-events check-types` fails with the above error. This was never caught in CI because when CI was refactored for Commonwealth end-to-end tests, `yarn --cwd packages/chain-events check-types` was removed.